dbot/dbot/Program.cs

```diff
@@ -38,6 +39,9 @@ public async Task MainAsync()
 
         public async Task InstallCommands() {
             client.MessageReceived += HandleCommand;
+
+            serviceCollection.AddSingleton(new NominationsService());
```

Comment:
I think this needs to be above before calling buildserviceprovider

Reply:
aite